src/directives/accessibility/focus-indicator/focus-indicator.directive.ts

```diff
@@ -11,7 +11,6 @@ import { FocusIndicatorService } from './focus-indicator.service';
 @Directive({
     selector: '[uxFocusIndicator]',
     exportAs: 'ux-focus-indicator',
-    providers: [LocalFocusIndicatorOptions]
```

Comment:
Can I just clarify why this was removed? Is it not needed any more?

Reply:
Yeah I had incorrectly added this in a previous commit - https://github.com/UXAspects/UXAspects/pull/1686/commits/f2e758e581ad7a60a41ce8c59a0113d6a518c320

Reply:
Perfect, thank you just wanted to make sure this wasn't unintentional